modules/protocol/src/vector.ts

```diff
@@ -199,7 +201,7 @@ export class Vector implements IVectorProtocol {
         }
         const method = "onReceiveProtocolMessage";
         const methodId = getRandomBytes32();
-        this.logger.debug({ method, methodId }, "Method start");
+        const label = logOperationTime(this.logger, method);
```

Comment:
We will likely want to keep the `methodId` to log throughout the other parts of the method so the logs are easily searchable. Maybe we should use the `label` as the `methodId` instead.

General purpose of the `methodId` is to get a unique strings so that you can easily identify all logs that came from a given method when searching in log dna or whatever other log parser

Reply:
Sure. This functionality is already internal to the `logOperationTime` method. I actually had it like that at first, but worried that not all methods would follow this standard of generating a methodId. Will add this back tho, thanks